catalyst/utils/tools/metric_manager.py

```diff
@@ -3,6 +3,7 @@
 from numbers import Number
 from time import time
 
+from catalyst.utils.distributed import distributed_mean
```

Comment:
could you please change the import so it would be only `from catalyst.utils import distributed_mean`